### DISCUSSION TITLE
Die erste seite, Verdrahtungsplan und links

### DISCUSSION THREAD

**Discussion by bf4ed** [hoylabs/OpenDTU-OnBattery#271] [General]
Bitte gibt auf die erste seite den kompleten verdrahtungsplan fur alle module. Display, CAN bus, ... alles was es unterstutzt.

Und da auf den verdrahtungsplan steht, 5V mod fur CAN modul, wo finde ich den mod?
Ich kann den link zu den mod nicht finden.

Irgendwie ist da sehr schwer was zu finden.

**Comment by spcqike** [comment]
Hilft dir das weiter? 

https://github.com/helgeerbe/OpenDTU-OnBattery/blob/master/docs/DeviceProfiles/nodemcu_esp32.json

Du kannst theoretisch die Pins frei verwenden. Musst dann halt in deinen device Profil angeben was wo ist. 

Ich verwende die in dem Profil hinterlegten Pins.

**Reply by bf4ed** [reply]
Sagen wir ich könnte damit was zusammenbasteln:)

Und wo finde ich denn 5V mod für das CAN modul?

**Reply by Manos1966** [reply]
Vielleicht hilft das:
Am Ende des Readme.md 
![failed](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/5e845ccc-c840-4a5f-acf2-95d20d8fc394)

**Reply by bf4ed** [reply]
Nein, alles was hier auf GitHub zu finden ist, ist nicht gut genug.

Aber ich hab es hier gefunden was ich brauchte:
https://forums.raspberrypi.com/viewtopic.php?t=141052

Und am ende sogar erfolgreich getestet, das Huawei  Netzteil mit CAN bus zu regeln.

Aber die ganze erste seite ist ein mess.
Jetzt hab ich ein anderes problem.

Wie teste ich das Relais für slot detect ob es sich uber OpenDTU einschaltet? Wie kann ich das manuell testen, oder einschalten? Und wann sollte es sich eigentlich einschalten in automatik betrieb?

**Reply by kloppy1984** [reply]
Wie schon oft festgestellt, ist die Elektrische Doku sehr Mangelhaft gemacht, Anschlüsse nicht vollständig, falsche Komponenten usw.

Da hilft sich nur alles selbst zusammen stellen.
Oder hoffen das jemand seinen Anschlussplan zeigt.

**Reply by helgeerbe** [reply]
@kloppy1984 Da gebe ich dir gerne recht. Es gibt halt keinen hauptberuflichen Entwickler, der für alle möglichen Komponenten ein qualitätsgeprüftes Handbuch bereitstellt.

Mit einfach meckern, machst du es dir zu einfach. Hier wird immer gerne geholfen, wenn jemand fragt.

Das Projekt lebt von der Community. Wer nicht entwickeln kann, aber trotzdem etwas beitragen möchte, kann auch sehr gerne die Dokumentation überarbeiten.

**Reply by kloppy1984** [reply]
Da ich Eltrektro Technik studiert habe, sollte das nicht so schwer für mich sein.

Ich kann es euch gern die Tage mal überarbeiten,  wenn ich irgendwo die Original Datei finde zum bearbeiten.
Zudem muss ich mal schauen wie ich dann Änderungen hochladen kann!

**Reply by helgeerbe** [reply]
Meine KiCad Files kann ich dir zur Verfügung stellen #138

**Comment by bf4ed** [comment]
Ich will ja das alle optionen auf einer Platine sind. Alle Module und sogar die zwei LED.

So was wie auf den Bildern. Den Prototyp muss ich noch zusammen löten, OLED kommt später nicht auf die Platine, ist nur zum testen. Und die Victron sachen hab ich nicht. Jetzt warte ich auf den Pylontech Akku das ich noch die CAN Verbindung teste.

![DSC_1661](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/130218342/f83c1225-8e31-43fe-8f8d-d3aac4e399cd)

![Screenshot_20230626-212012_cropped](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/130218342/14705ec6-9ad1-417d-a5cd-54ade3981f2c)

**Reply by kloppy1984** [reply]
Ich kann meine Platine in Fusion360 erweitern, eine RJ45 Buchse für den Pylontech Akku ist schon vorhanden. Ein LCD mit 4 Pins Anschluss kann ich gern auch drauf machen.

Ich befürchte das dein Mini ESP mit den Pins nicht ausreichend ist. 
Victron habe ich da, Pylontech kann ich mir leihen.
Ansteuerung für das 5V Relais ist auch machbar

**Reply by bf4ed** [reply]
Es kommt nicht der mini. Das ist nur was ich gehabt hab. Auf den zweiten Foto ist ein anderer ESP32 mit mehr pins. Zuerst muss ich alles testen und dann könnte man die Platine in KiCAD Zeichen.
